velox/py/plan_builder/PyPlanBuilder.h

```diff
@@ -141,13 +141,25 @@ class PyPlanBuilder {
   ///    input_files=[PARQUET("my_file.parquet")],
   ///  )
   PyPlanBuilder& tableScan(
-      const velox::py::PyType& outputSchema,
+      const PyType& outputSchema,
       const pybind11::dict& aliases,
       const pybind11::dict& subfields,
       const std::string& rowIndexColumnName,
       const std::string& connectorId,
       const std::optional<std::vector<PyFile>>& inputFiles);
 
+  /// Adds a table writer node to write to an output file(s).
+  ///
+  /// @param outputSchema The schema to be used when writing the file (columns
+  /// and types).
+  /// @param connectorId The id of the connector to use during the write
+  /// process.
+  /// @param outputFile The output file to be written.
```

Comment:
nit: the order of the params in the comments don't match the order of the arguments to the function